Xamarin.Forms.Core/Properties/GlobalAssemblyInfo.cs

```diff
@@ -1,6 +1,35 @@
 using System.Reflection;
 
+#if DEBUG
```

Comment:
AssemblyConfiguration is also emitted automatically from the current project `$(Configuration)`: https://github.com/dotnet/sdk/blob/master/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.GenerateAssemblyInfo.targets#L70-L72